app/models/guests_room.rb

```diff
@@ -0,0 +1,4 @@
+class GuestsRoom < ApplicationRecord
```

Comment:
We typically name join models with singular - GuestRoom - and the corresponding table has a plural but only for the second resource - guest_rooms.